Front-end/parquick/src/components/Map/Map.jsx

```diff
@@ -0,0 +1,77 @@
+//
+import * as React from 'react'
+import { GoogleMap, useLoadScript, Marker} from '@react-google-maps/api'
+import './Map.css';
+import { useCallback } from 'react';
+import { useRef } from 'react';
+import { useMemo } from 'react';
+import DriverLocation from '../DriverLocation/DriverLocation';
+import { useState } from 'react';
+import { useEffect } from 'react';
+
+
+
+
+function Map(): React.MixedElement {
+
+
+    const [location, setLocation] = useState();
+
+
+    const mapRef = useRef();
+    const center = useMemo(() => ({ lat: 37.4800384, lng: -122.1558272 }), []);
```

Comment:
Let's avoid "magic numbers" embedded in code.
Better to declare them above outside the component or in a general constants file so that they have names such as `MAP_DEFAULT_LAT` and `MAP_DEFAULT_LNG`.

Reply:
Got it, thank you!